README.md

```diff
@@ -46,31 +36,26 @@ also be changed.
 
 * Update the template constants (install dates and deadlines) in config/template_values.yml.
 
-## Initialize the Database
```

Comment:
I did cover that in the updated getting started section: https://github.com/ywwg/ffagc/pull/61/files#diff-04c6e90faac2675aa89e2176d2eec7d8R12